src/workarounds/forwarddiff_rules.jl

```diff
@@ -180,15 +180,31 @@ function self_consistent_field(basis_dual::PlaneWaveBasis{T};
 
     ## Implicit differentiation
     hamψ_dual = ham_dual * ψ_dual
-    δHψ = [ForwardDiff.partials.(δHψₖ, 1) for δHψₖ in hamψ_dual]
-    δψ, response = solve_ΩplusK(basis, ψ, -δHψ, occupation;
-                                tol_cg=scfres.norm_Δρ, verbose=response.verbose)
-    δρ  = compute_δρ(basis, ψ, δψ, occupation)
+    N = ForwardDiff.npartials(T)
+    δresults = ntuple(N) do j
+        δHψj = [ForwardDiff.partials.(δHψₖ, j) for δHψₖ in hamψ_dual]
+        δψj, responsej = solve_ΩplusK(basis, ψ, -δHψj, occupation;
+                                      tol_cg=scfres.norm_Δρ, verbose=response.verbose)
+        δρj = compute_δρ(basis, ψ, δψj, occupation)
+        δψj, δρj, responsej
+    end
+    δψ = [δψj for (δψj, δρj, responsej) in δresults]
+    δρ = [δρj for (δψj, δρj, responsej) in δresults]
+    response = [responsej for (δψj, δρj, responsej) in δresults]
 
     ## Convert, combine and return
     DT = ForwardDiff.Dual{ForwardDiff.tagtype(T)}
-    ψ_out = [Complex.(DT.(real(ψₖ), real(δψₖ)), DT.(imag(ψₖ), imag(δψₖ))) for (ψₖ, δψₖ) in zip(ψ, δψ)]
-    ρ_out = DT.(scfres.ρ, δρ)
+    ψ_out = map(ψ, δψ...) do ψₖ, δψₖ...
```

Comment:
I'd rathe we typed the k in plain ASCII (that's what we do in several other places)

Reply:
Acutally ... I'm not sure that's such a bad idea. I find the subscript k rather readable.

Reply:
I don't really like unicode subscripts. It's rather gimmicky, and some fonts just don't support it. The place where I agree it makes sense is for longer subscripts, like x_{n+1}. Here ψk is not really worse than ψₖ... Also if we want to do it we should do it everywhere

Reply:
yes sure, consistency is clear.

Reply:
ok let's leave it for now